Refresh.GameServer/Types/Levels/CustomRewards/CustomReward.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Xml.Serialization;
+using Realms;
+
+namespace Refresh.GameServer.Types.Levels.CustomRewards;
+
+[XmlType("customReward")]
+public partial class CustomReward : IEmbeddedObject
+{
+    [XmlAttribute(AttributeName = "id")] public int Id { get; set; }
```

Comment:
I feel as if this should use the sequential ID system

Reply:
Nevermind, this is an embedded object. I see now.

Reply:
Honestly, I don't really know what I'm doing here. The reason I put the Id parameter here is only because the publish request includes the Ids.

```
<customRewards>
	<customReward id="0">
		<enabled>true</enabled>
		<description>50 score</description>
		<condition>score</condition>
		<amountNeeded>5.000000</amountNeeded>
	</customReward>
	<customReward id="1">
		<enabled>true</enabled>
		<description>5 lives lost</description>
		<condition>lives</condition>
		<amountNeeded>5.000000</amountNeeded>
	</customReward>
	<customReward id="2">
		<enabled>true</enabled>
		<description>30 second timer</description>
		<condition>time</condition>
		<amountNeeded>899.999939</amountNeeded>
	</customReward>
</customRewards>
```

Reply:
Sorry, didn't see your response. Forgot to refresh lmfao

Reply:
Yeah, probably fine to just leave it